README.md

```diff
@@ -38,55 +34,46 @@ tools installed.
-good to be merged. It is based on a script called `go`, which is implemented using Ruby and Rake. Always run the `go` script before committing or pushing to GitHub.
-To run the build:
+### Installation
 
+To install dependencies:
 ```shell script
-./go
+npm install
 ````
 
-By default, the build process fetches all dependencies, compiles, lints, 
-formats and tests the codebase. If the linting or formatting tasks find problems, the script will attempt to fix them silently, so always check for changes in your files before committing or pushing to GitHub.
-There are also tasks for each step that can be run separately. This and
-subsequent sections provide more details of each of the tasks.
+---
+### Configuration
 
-To fetch dependencies:
+Before you can start interacting with the contracts, via hardhat, please make a copy of the `.env.example` file and call it `.env`.
 
-```shell script
-./go dependencies:install
-```
+---
+### Build
 
-To compile the contracts:
+All of the available commands can be found in `package.json`.
 
+To compile:
 ```shell script
-./go contracts:compile
-```
+npm run contracts:compile
+````
 
 ---
 ### Run
-To deploy instances of the contracts for local development without prior knowledge of Hardhat, first copy .env.example to .env and run the following command:
+
 ```shell
 ./go contracts:run
 ```
 
-This command starts up built-in Hardhat Network and migrates all contracts to the Hardhat Network instance. The .env file has a hard-coded value for the BOSON_TOKEN address, which points to account #9 of the local hardhat network. This isn't an actual
-BOSON token address but just a valid address that will allow the deployment scripts to work. It's not necessary to deploy a BOSON token to run the unit tests locally (see Unit Tests section), as the unit tests deploy their own contract instances.
+*Note that*: This command starts up built-in Hardhat Network and migrates all contracts to the Hardhat Network instance. The `.env` file has a hard-coded value for the `BOSON_TOKEN` address, which points to account #9 of the local hardhat network. This isn't an actual BOSON token address but just a valid address that will allow the deployment scripts to work. It's not necessary to deploy a BOSON token to run the unit tests locally (see Unit Tests section), as the unit tests deploy their own contract instances.
 
-If preferred by those who are familiar with Hardhat, the standard Hardhat commands can be used. Ganache can be started up manually by configuring a local network to be run against or using the `hardhat-ganache` plugin or you could start a Hardhat Network using `npx hardhat node`. For more information on how this can be achieved refer to the [official Hardhat documentation](https://hardhat.org/guides/ganache-tests.html#running-tests-with-ganache)
+If preferred by those who are familiar with Hardhat, the standard Hardhat commands can be used.
 
 In a separate terminal, contracts can be deployed using
```

Comment:
If you just want to deploy to the built-in hardhat node `--network localhost` is not needed. You should only have to run` npx hardhat deploy`. A separate terminal and `--network localhost` is only needed if the `npx hardhat node` command is used.

Reply:
thanks, made some changes to the doc